bcdaworker/queueing/manager/que_test.go

```diff
@@ -133,10 +133,10 @@ func TestProcessJobFailedValidation(t *testing.T) {
 		expectedErr error
 		expLogMsg   string
 	}{
-		{"ParentJobCancelled", worker.ErrParentJobCancelled, nil, `^queJob \d+ associated with a cancelled parent Job`},
-		{"NoBasePath", worker.ErrNoBasePathSet, nil, `^Job \d+ does not contain valid base path`},
-		{"NoParentJob", worker.ErrParentJobNotFound, repository.ErrJobNotFound, `^No job found for ID: \d+ acoID.*Will retry`},
-		{"NoParentJobRetriesExceeded", worker.ErrParentJobNotFound, nil, `No job found for ID: \d+ acoID.*Retries exhausted`},
+		{"ParentJobCancelled", worker.ErrParentJobCancelled, nil, `^Removing queuejob from que; parent job \d+ cancelled.`},
```

Comment:
Simplified log messages; no longer using the interpolation

Reply:
I tried to move any fields that are an identifier out of the `msg` string field and into their own fields; it seemed redundant to list it again in the message! 😄

Reply:
that's a good point -- let me do some looking through splunk to make sure no reports break with these changes.